sim/core/debuffs.go

```diff
@@ -33,6 +33,10 @@ func applyDebuffEffects(target *Unit, targetIdx int, debuffs *proto.Debuffs, rai
 		}
 	}
 
+	if debuffs.JudgementOfTheCrusader && targetIdx == 0 {
+		MakePermanent(JudgementOfTheCrusaderAura(target, level, 1, 0))
```

Comment:
The Multi can be implemented using a TristateEffect.

Reply:
Changed it.